tkg/client/management_components.go

```diff
@@ -101,6 +102,30 @@ func (c *TkgClient) InstallOrUpgradeManagementComponents(kubeconfig, kubecontext
 	return err
 }
 
+func correctAddonsManagerPackageVersion(packageVersion string) string {
+	envDefinedVersion := os.Getenv("_ADDONS_MANAGERR_PACKAGE_VERSION")
```

Comment:
Also can we add this in getAddonsManagerPackageVersion ? Getting the env var in correctAddonsManagerPackageVersion is confusing.

Reply:
really? it makes more sense to me that we look at "_ADDONS_MANAGER_PACKGE_VERSION" in a function that obverrides addonsManagerPackage version. Doint it here will also make it esier to roll back. I am trying to get a certain level of indirection to to be so pervasive in the code.

Reply:
yeap.